exercises/react/.meta/plugins.py

```diff
@@ -0,0 +1,20 @@
+def add_callbacks(case):
+    observers = []
+    callbacks = []
+    operations = list(filter(lambda op: op["type"] == "add_callback", case["input"]["operations"]))
```

Comment:
The list comprehension form would be shorter here: `[op for op in case["input"]["operations"] if op["type"] == "add_callback"]`.